app/views/jobseekers/profiles/qualified_teaching_status/edit.html.slim

```diff
@@ -0,0 +1,23 @@
+- content_for :page_title_prefix, t(".page_title")
+
+- content_for :breadcrumbs do
+  = govuk_back_link text: t("buttons.back"), href: jobseekers_profile_path
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    = form_for @form, url: jobseekers_profile_qualified_teaching_status_path, method: "patch" do |f|
+      = f.govuk_error_summary
+
+      span.govuk-caption-l = t(".page_title")
+
+      = f.govuk_radio_buttons_fieldset :qualified_teaching_status, legend: { size: "l", tag: "h1" } do
+        = f.govuk_radio_button :qualified_teaching_status, "yes", link_errors: true
+          = f.govuk_text_field :qualified_teaching_status_year,
+            label: { text: "Year QTS awarded", class: "govuk-label--s" }
```

Comment:
Think you can add this label to `activerecord.yml` instead. That's the pattern we follow elsewhere.